providers/activedir/activedirProvider.go

```diff
@@ -21,7 +21,7 @@ type adProvider struct {
 // Register with the dnscontrol system.
 //   This establishes the name (all caps), and the function to call to initialize it.
 func init() {
-	providers.RegisterDomainServiceProviderType("ACTIVEDIRECTORY_PS", newDNS)
+	providers.RegisterDomainServiceProviderType("ACTIVEDIRECTORY_PS", newDNS, providers.CanUseSRV)
```

Comment:
I am skeptical any of these providers are going to work as-is. They will need to read the weight/priority and other fields from the providers and marshal it into our struct format. And vice-versa. Maybe that bit is still WIP?